extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js

```diff
@@ -5,11 +5,15 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.veles', [
 	'ext.wikia.adEngine.video.vastUrlBuilder',
 	'wikia.geo',
 	'wikia.instantGlobals',
+	'wikia.log',
 	'wikia.window'
-], function (adContext, prebid, vastUrlBuilder, geo, instantGlobals, win) {
+], function (adContext, prebid, vastUrlBuilder, geo, instantGlobals, log, win) {
 	'use strict';
 
 	var bidderName = 'veles',
+		loggerEndpoint = '/wikia.php?controller=AdEngine2Api&method=postVelesInfo',
+		loggerSampling = 0.01,
```

Comment:
This variable is very unobvious, you need to know Math.random result range to know a proportion of the sampling